imports/api/transactions/payments/payments.js

```diff
@@ -200,6 +201,7 @@ Transactions.categoryHelpers('payment', {
       if (unallocatedAmount === 0) return false;
       const bill = Transactions.findOne(billPaid.id);
       if (!bill.isPosted()) throw new Meteor.Error('Bill has to be posted first');
+      debugAssert(billPaid.amount < 0 === bill.amount < 0, 'Bill amount and its payment must have the same sign');
```

Comment:
Ha a UI-on a user be tud vinni ilyen adatot, es szolni kene neki, akkor productionAssert kell ide. A debugAssert az a developer-nek szol, es bug felfedezesre alkalmazzuk